cms/settings/base.py

```diff
@@ -548,6 +555,16 @@
 # that let us use SMTP.
 # https://docs.djangoproject.com/en/2.1/topics/email/
 
+EMAIL_BACKEND = env.get("EMAIL_BACKEND", "django.core.mail.backends.console.EmailBackend")
+FROM_EMAIL = env.get("FROM_EMAIL", "example@mail.com")
+
+if EMAIL_BACKEND == "django_ses.SESBackend":
```

Comment:
q: what about adding https://github.com/django-ses/django-ses to pyproject.toml / update poetry.lock?

Reply:
hold fire on this as per https://github.com/ONSdigital/dis-wagtail/pull/210#discussion_r2120785872